crates/lib3h/src/transport/memory_mock/ghost_transport_memory.rs

```diff
@@ -63,6 +103,52 @@ impl GhostTransportMemory {
             ),
             connections: HashSet::new(),
             maybe_my_address: None,
+            last_discover: start,
+            discover_interval_ms: interval as u128,
+        }
+    }
+
+    fn try_discover(&mut self) {
+        if let Some(t) = self.last_discover {
```

Comment:
This code block is a bit overly nested but probably not worth cleaning up at this point.